src/server/ConDrvDeviceComm.cpp

```diff
@@ -150,3 +150,15 @@ DeviceComm::~DeviceComm()
 
     return S_OK;
 }
+
+[[nodiscard]] ULONG_PTR ConDrvDeviceComm::PutHandle(const void* handle)

```

Comment:
nit: doc comments, technically, as the rest of this file has 'em.

Reply:
reckon i oughtta

Reply:
Docced.